coap-gateway/service/refreshToken_test.go

```diff
@@ -68,7 +68,7 @@ func TestRefreshTokenHandlerWithRetry(t *testing.T) {
 
 	testRefreshToken := testEl{
 		name: "Retry",
-		in:   input{coapCodes.POST, `{"di": "` + CertIdentity + `", "uid":"` + AuthorizationUserId + `", "refreshtoken":"123"}`, nil},
+		in:   input{coapCodes.POST, `{"di": "` + CertIdentity + `", "uid":"` + AuthorizationUserId + `", "refreshtoken":"refresh-token"}`, nil},
```

Comment:
Does this mean that we always get "refresh-token" as RefreshToken regardless of the input?

Reply:
Regardless of what kind of input?
Yes. A refresh token with value `"refresh-token"` is returned every time. A random value is generated only if you set `refreshTokenRestrictionLifetime` to other then default value.

Reply:
This input:
```
input{coapCodes.POST, `{"di": "` + CertIdentity + `", "uid":"` + AuthorizationUserId + `", refreshtoken":"refreshtoken"}
```
I guess it previously automatically returned as RefreshToken what was in the `refreshtoken` value in the input, but now it returns `refresh-token` (unless `refreshTokenRestrictionLifetime` is set). The test is still valid, so this is OK, I guess I'm just confused about how does our mock oauth server work, it's beginning to be complicated. :D

Reply:
Hm, I am not aware of this functionality - set in the request to mock OAuth Server value of the refresh token which it then returns.